doc/OnlineDocs/contributed_packages/pyros.rst

```diff
@@ -397,22 +397,23 @@ If we choose to designate all variables as either design or state variables, wit
   ...                                  options = {
   ...                                     "objective_focus": pyros.ObjectiveType.worst_case,
   ...                                     "solve_master_globally": True
-  ...                                   })
+  ...                                   }) # doctest: +ELLIPSIS
   ===========================================================================================
   PyROS: Pyomo Robust Optimization Solver ...
   ===========================================================================================
   ...
   INFO: Robust optimal solution identified. Exiting PyROS.
+  INFO: Objective: 4836738...
   ...
 
   >>> # === Print results ===
   >>> single_stage_final_objective = pyo.value(results_1.final_objective_value)
-  >>> print("Final objective value: %s" % single_stage_final_objective) # doctest: +ELLIPSIS
+  >>> print("Final objective value: %s" % single_stage_final_objective)
   Final objective value: ...
```

Comment:
Can you change this to
```
>>> print("Final objective value: %d" % single_stage_final_objective)
Final objective value: 48367381
```
...and if the objective is slightly noisy, you can just round it (e.g., `round(single_stage_final_objective, -1)`